app/javascript/src/Policies/util.jsx

```diff
@@ -35,10 +36,45 @@ function generateGuid (): string {
   return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4() + '-' + s4() + s4() + s4()
 }
 
+function parsePolicies (registry: RawRegistry): Array<RegistryPolicy> {
+  let policies: Array<RegistryPolicy> = []
+  for (let key in registry) {
+    registry[key].forEach(policy => policies.push(parsePolicy(key, policy)))
+  }
+  return policies
+}
+
+function parsePolicy (key: string, policy: RawPolicy): RegistryPolicy {
+  return Object.assign({}, policy,
```

Comment:
Why not using spread syntax?
```js
return { ...policy, name: key, humanName: policy.name, data: {} }
```

Reply:
This was migrated from another file. I'll change it anyways